### ISSUE TITLE
Assignment 3.3 is unclear

### ISSUE THREAD

**Issue by jackodirks** [tu-os-course/TI2726C-Assignments-2016#6]
The description of assignment three says the following:

Write a multi-threaded program that calculates various statistical values for a list of numbers. This program will be passed a series of numbers on the command line and will then create three separate worker threads. One thread will determine the average of the numbers, the second will determine the maximum value, and the third will determine the minimum value. For example, suppose your program is passed the integers:
...
The average value is 82
The minimum value is 72
The maximum value is 95
The median value is X
The standard deviation is Y

It is not clear which thread calculates the median and the standard deviation.

**Comment by ajrouvoet** [comment]
I totally agree with your assessment. As such I can't really fail your for any solutions that isn't clearly wrong or not-as-intended. Meaning that the minimum requirement is that  we see _3 worker threads_ handling _5 tasks_. 

I'd love to see that as soon as a thread finishes, a new one is started with the next "task in the queue". Then you're working towards schedulers.

**Comment by ajrouvoet** [comment]
Note to self: this should be fixed in the manual
